tests/e2e/workload-lifecycle/services/java-http-server/java-supported-docker-env.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM maven:3.8.5-openjdk-17 AS build
+COPY src /home/app/src
+COPY pom.xml /home/app
+RUN mvn -f /home/app/pom.xml clean package
+
+FROM eclipse-temurin:17-jre-jammy
+ENV JAVA_OPTS="-Dthis.does.not.exist=true"
```

Comment:
Where do we assert that this value indeed exists in the application?
If odigos remove it when env overwriting, will the test fail?